src/pages/go/adding-alerts-and-slos.mdx

```diff
@@ -0,0 +1,49 @@
+import { Callout, Steps, Tabs, Tab } from "nextra-theme-docs";
+import AlertsSLOsLangIntro from "@/snippets/adding-alerts-slos.mdx";
+
+<AlertsSLOsLangIntro language="Go"/>
+
+## Pre-requisites
+
+1. Make sure you have setup and initialized Autometrics libraries as described in [Quickstart section](quickstart).
+
+2. Add the pre-configured Prometheus rules file to your Prometheus configuration. This file is located in the [Autometrics shared repository](https://github.com/autometrics-dev/autometrics-shared/blob/main/autometrics.rules.yml)
+
+## Usage
+
+Taking our example Autometrics-instrumented function from earlier:
+
+```go filename="main.go"
+//autometrics:doc
```

Comment:
The API changed in a version that's not published yet, so not sure how to deal with that (I don't know when I'll have time to make a release yet, maybe next week)

```suggestion
//autometrics:inst
```